tests/test_default_gaussian.py

```diff
@@ -534,6 +535,55 @@ def test_reduced_state(self):
         self.assertAllAlmostEqual(res[0], expected[0], delta=self.tol)
         self.assertAllAlmostEqual(res[1], expected[1], delta=self.tol)
 
+    def test_samples_coherent(self):
+        """Test that sampled values are calculated correctly for a coherent state by comparing with first and second moments"""
+        self.logTestName()
+        n_samples = 1000000

```

Comment:
With the tolerance now depending on the number of samples I would suggest we decrease this to a more reasonable value of, say, 10000 for better performance.